app/services/new_version_service.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# Duplicates a version, creating new related objects rather than point at objects for a past version.
+# This is like NewVersionParameter, but it works on objects instead of parameters.
+class NewVersionService
+  # @param [WorkVersion] old_version
+  # @return [WorkVersion] the new version
+  def self.dup(old_version, increment_version: false, save: false, version_description: nil, state: nil)
```

Comment:
I don't understand why all these options are needed. Isn't there only one place where this is being used?